external/settings.php

```diff
@@ -7,6 +7,12 @@
 
 $tmpl = new OCP\Template( 'external', 'settings');
 
-$tmpl->assign('images', glob(\OC_App::getAppPath('external') . '/img/*.*'));
+$images = glob(\OC_App::getAppPath('external') . '/img/*.*');
+$theme = \OC_Util::getTheme();
```

Comment:
Better add a check whether `$themes` is empty before doing the `file_exists` call.
Sometimes strange people move their folders around in a way where it might even pass, if you have a folder "themes/apps/external/img". I'd rather not take that risk :smile: